### PR TITLE
Update coing_gecko.go add data point marketCapRank (#1)

### PR DESCRIPTION
**Pull request by Cornelivsd**
* Update coing_gecko.go add data point marketCapRank

Adding market cap rank data point for use in nickname sorting

### CODE REVIEW

File: utils/coin_gecko.go [hunk 1]
```diff
@@ -31,14 +31,17 @@ type MarketData struct {
...
 type GeckoPriceResults struct {
-	ID         string     `json:"id"`
-	Symbol     string     `json:"symbol"`
-	Name       string     `json:"name"`
-	MarketData MarketData `json:"market_data"`
+	ID            string     `json:"id"`
+	Symbol        string     `json:"symbol"`
+	Name          string     `json:"name"`
+	MarketData    MarketData `json:"market_data"`
+	MarketCapRank int64    'json:"market_cap_rank"'
```
Comment (rssnyder) [inline utils/coin_gecko.go:44]:
```suggestion
	MarketCapRank int64       `json:"market_cap_rank"`
```

**Comment by Cornelivsd** [comment]
Please kick it back so I can convert to draft if still possible.

**Comment by rssnyder** [comment]
bad bot

**Review by Vladisvob — APPROVED** [review]

**Review by Vladisvob — APPROVED** [review]
#109

**Pull request closed without merging**